entities/package.py

```diff
@@ -0,0 +1,97 @@
+from typing import List
+from common import format_float
+from .common import Name, Description, Keywords, Author, Version, Created, Category, Deprecated, Position, Rotation, Polygon, Layer, Height, Align, Mirror, Value, FloatValue, EnumValue, BoolValue
+
+
+class Pad():
+    def __init__(self, uuid: str, name: Name):
+        self.uuid = uuid
+        self.name = name
+
+
+class StrokeWidth(FloatValue):
+    def __init__(self, stroke_width: float):
+        super().__init__('stroke_width', stroke_width)
+
+
+class LetterSpacing(EnumValue):
+    AUTO = 'auto'
+
+    def get_name(self) -> str:
+        return 'letter_spacing'
+
+
+class LineSpacing(EnumValue):
+    AUTO = 'auto'
+
+    def get_name(self) -> str:
+        return 'letter_spacing'
+
+
+class AutoRotate(BoolValue):
+    def __init__(self, auto_rotate: bool):
+        super().__init__('auto_rotate', auto_rotate)
+
+
+class StrokeText():
+    def __init__(self, uuid: str, layer: Layer, height: Height, stroke_width: StrokeWidth, letter_spacing: LetterSpacing, line_spacing: LineSpacing, align: Align, position: Position, rotation: Rotation, auto_rotate: AutoRotate, mirror: Mirror, value: Value):
+        pass
+
+
+class Side(EnumValue):
+    THT = 'tht'
+
+    def get_name(self) -> str:
+        return 'side'
+
+
+class Shape(EnumValue):
+    ROUND = 'round'
+    RECT = 'rect'
+
+    def get_name(self) -> str:
+        return 'shape'
+
+
+class Size():
+    def __init__(self, width: float, height: float):
+        # TODO: Are width and height good names?
+        self.width = width
+        self.height = height
+
+    def __str__(self) -> str:
+        return '(size {} {})'.format(format_float(self.width), format_float(self.height))
+
+
+class Drill(FloatValue):
+    def __init__(self, drill: float):
+        super().__init__('drill', drill)
+
+
+# TODO: Naming:
+class FootprintPad():
```

Comment:
I'm not sure how I should name this two classes of `Pad`. One is only the uuid and name and the others are used in the footprint and contains much more.

Reply:
Maybe PadDef for the simple one?

Reply:
Or `PadRef`?

Reply:
Works too.

Reply:
In LibrePCB they are called `PackagePad` and `FootprintPad`...

Reply:
Perfect. Done.